docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,127 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. This includes creating a GitHub application, so {ProductName} can access users' pull requests, and configuring a smee channel, to listen for those PRs. The first procedure in this document explains how to perform these tasks. 
```

Comment:
smee channel is needed in the context where konflux is running on a local setup that cannot expose the endpoint that trigger the pipeline outside of the cluster.

If this document should be more generic then I think it should not state smee as default.